README.md

```diff
@@ -1,2 +1,70 @@
 # comp110-worksheet-C
+<<<<<<< HEAD
+Base repository for COMP110 worksheet C (Arturs Zingis)
+
+
+**Algorithm:**
+
+	procedure HASDUPLICATE(list)
+	let n be the length of list
+	for i = 0, 1,..., n - 1 do
+	for j = 0, 1,..., n - 1 do
+	if i =/ j and list[i] = list[j] then
+	return true
+	end if
+	end for
+	end for
+	return false
+	end procedure
+
+
+A) O(n). This algorithm sets two lists with i/j and for every item in the list, it takes away 1 from the length of the list and then checking if both lists are equal and if they are then return true and if not, return false. 
+
+B) It would be the worst case scenario because we would have go through the list n times and with each iteration, you would have to go through the first loop. Which would take double the time.
+
+**Algorithm 2**
+
+	procedure HASDUPLICATE(list)
+	let n be the length of list
+	for i = 0, 1,..., n - 1 do
+	for j = 0,..., i - 1 do
+	if i =/ j and list[i] = list[j] then
+	return true
+	end if
+	end for
+	end for
+	return false
+	end procedure
+
+c) This loop will carry out the same operation but instead of referring to the number of lists in the (j), it will link back to the first (i) list. The result of the (i) list -1 will determine (j).
```

Comment:
Not sure what you mean here. I was looking for something like: the first algorithm compares each pair of elements twice (i.e. A==B and B==A), whereas the second algorithm eliminates the redundant checks